website/source/docs/enterprise/read-scale/index.html.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: "docs"
+page_title: "Consul Enterprise Enhanced Read Scalability"
+sidebar_current: "docs-enterprise-read-scale"
+description: |-
+  Consul Enterprise supports increased read scalability without impacting write latency.
+---
+
+# Consul Enterprise Enhanced Read Scalability
+
+In [Consul Enterprise](https://www.hashicorp.com/consul.html), servers can be
+explicitly marked as non-voters. Non-voters will receive the replication stream
+but will not take part in quorum (required by the leader before log entries can
+be committed). Adding explicit non-voters will [scale
+reads](https://www.consul.io/docs/guides/autopilot.html#server-read-scaling)
```

Comment:
When linking to something internal, try to use relative URLs instead, so consider changing this to:

```
/docs/guides/autopilot.html#server-read-scaling
```

This is for two reasons:

1. In local dev, the links still work
1. Modern browsers will render that page faster because it doesn't need to lookup the DNS